cmd/gen-go-sample/inittree.go

```diff
@@ -119,6 +119,9 @@ func (t *initTree) get(k string, info pbinfo.Info) (*initTree, error) {
 							// key is always a primitive type
 							v.typ.keyType = &initType{prim: f2.GetType()}
 						case "value":
+							// value is a primitive type
+							v.typ.valueType = &initType{prim: t2}
```

Comment:
t2 := f2.GetType(), remove the variable init in the `if` and use it there too

Reply:
(Tried to edit the file using directory in github, turns out that's a bad decision.)